composer.json

```diff
@@ -11,8 +11,13 @@
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "guzzlehttp/psr7": "^1.0",
+        "lstrojny/fxmlrpc": "^0.14.0",
+        "php-http/guzzle6-adapter": "^1.1",
```

Comment:
You should not rely on a specific adapter. This must be a user choice.

Ref: http://docs.php-http.org/en/latest/httplug/library-developers.html